lib/simplecov.rb

```diff
@@ -220,7 +221,7 @@ def process_result(result, exit_status)
       if result_exit_status == SimpleCov::ExitCodes::SUCCESS # No result errors
         write_last_run(covered_percent)
       end
-      result_exit_status
+      final_result_process? ? result_exit_status : SimpleCov::ExitCodes::SUCCESS
```

Comment:
I wonder if this could be simplified. I guess the thinking here is that we'll always say it's success unless we were the last process? Might be nice to say in a comment or something.